cl/search/management/commands/import_harvard_pdfs.py

```diff
@@ -0,0 +1,220 @@
+import json
+import logging
+import os
+import tempfile
+
+import boto3
+from botocore.exceptions import BotoCoreError, ClientError
+from django.conf import settings
+from django.core.files.base import ContentFile
+from django.core.files.storage import default_storage
+from django.core.management.base import BaseCommand
+from storages.backends.s3boto3 import S3Boto3Storage
+from tqdm import tqdm
+
+from cl.lib.storage import IncrementingAWSMediaStorage
+from cl.search.models import OpinionCluster
+
+logger = logging.getLogger(__name__)
+
+
+class HarvardPDFStorage(S3Boto3Storage):
```

Comment:
Maybe this could be in cl/lib/storage.py, what do you think @flooie?

Reply:
Yeah - I think we should move anything storage related together

Reply:
Makes sense, moved that to storage.py